plugins/woocommerce/package.json

```diff
@@ -122,7 +116,10 @@
 						"tests/php/**/*.php",
 						"tests/legacy/unit-tests/**/*.php",
 						"tests/unit-tests/**/*.php",
-						".wp-env.json"
+						".wp-env.json",
+						"woocommerce.php",
```

Comment:
I was wondering if there is a case, where we won't be triggering CI for a change in `*.php` file across the project?

Pehaps maybe we can consider just tirggering CI for `**/*.php` and remove the rest of the paths?

Reply:
Updated with [f3f23ea](https://github.com/woocommerce/woocommerce/pull/50399/commits/f3f23eafee37cc1701aa7aae5818374954da6ad4)